src/client/mapblock_mesh.h

```diff
@@ -125,6 +203,15 @@ class MapBlockMesh
 			m_animation_force_timer--;
 	}
 
+	/// update transparent buffers to render towards the camera
+	void updateTransparentBuffers(const v3f &camera_pos, const v3s16 &block_pos);
+	void consolidateTransparentBuffers();
+
+	/// get the list of transparent buffers
+	std::vector<PartialMeshBuffer> &getTransparentBuffers()
```

Comment:
could be const

Reply:
It's now const along with a bunch of other methods.